prime-dotnet-webapi/Migrations/20220106014622_AddCollegeForPlrRoleType.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Prime.Migrations
+{
+    public partial class AddCollegeForPlrRoleType : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "CollegeForPlrRoleType",
```

Comment:
We have two migrations adding this table causing error 'relation "CollegeForPlrRoleType" already exists'. You may need to add both migrations as one.